src/applications/gi-sandbox/components/profile/BenefitsForm.jsx

```diff
@@ -108,6 +110,7 @@ export class BenefitsForm extends React.Component {
               text: 'Which GI Bill benefit do you want to use?',
               modal: 'giBillChapter',
               ariaLabel: ariaLabels.learnMore.giBillBenefits,
+              buttonId: createId('gi bill benefits learn more'),
```

Comment:
`createId` is useful when you want to pass in something like the text value to become the id
Why not?
```suggestion
              buttonId: 'gi_bill_benefits_learn_more',
```

Reply:
Or do and append the "learn more" in the `renderLearnMoreLabel`
```suggestion
              buttonId: 'gi bill benefits',
```

Reply:
Yeah I dont think the majority of these need the `createId`, I can pull them out and use text values.